src/components/Setevent/Setevent.tsx

```diff
@@ -0,0 +1,19 @@
+import styles from './Setevent.module.scss';
+import { ReactNode, PropsWithChildren } from 'react';
+
+interface SetEventProps {
+  header: ReactNode;
+  footer: ReactNode;
+}
+
+export default function Setevent(props: PropsWithChildren<SetEventProps>) {
+  return (
+    <>
```

Comment:
no need? you already have one single div inside

Reply:
only when you have multiple elements on the same level, e.g.

```
<>
  <A />
  <B />
<>
```

^ that is because React always needs one element to attach some children to it. So, when you have multiple you need to wrap into `<></>` (called Fragment)

In this case, there's only one child in it so it doesn't make sense to wrap into a Fragment since it could be the child itself.

Reply:
I understood very well